.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+## Description
+
+Please include a summary of the changes made.
+
```

Comment:
also maybe add a closes # (issue)

Reply:
For now we don’t use issues, nice adding for future